lesson03/assignment/index.html

```diff
@@ -4,14 +4,13 @@
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<meta http-equiv="X-UA-Compatible" content="ie=edge">
-	<title>Fun with Constructors!</title>
+	<title>Lesson03 with Constructors!</title>
   <link rel="stylesheet" href="./css/style.css">
-	<script src="index.js"></script>
+	<!-- <script src="index.js"></script> -->
 </head>
 <body>
-	<h1>Assignment 3</h1>
-	<section class="template-hook">
-		Fill this section with the templated components
-	</section>
+	<main class="main">
```

Comment:
Semantic elements (like main, footer, etc) help remove the need for boilerplate classes like this.